runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -6534,7 +6534,8 @@ done:;
 		U_16 cpIndex = *(U_16 *)(J9ROMCLASS_SPECIALSPLITMETHODREFINDEXES(clazz->romClass) + splitTableIndex);
 		J9RAMSpecialMethodRef *ramMethodRef = (J9RAMSpecialMethodRef*)ramConstantPool + cpIndex;
 		/* argCount was initialized when we initialized the class (i.e. it is non-volatile), so no memory barrier is required */
-		if (0 == _sp[ramMethodRef->methodIndexAndArgCount & 0xFF]) {
+		j9object_t receiver = ((j9object_t *)_sp)[ramMethodRef->methodIndexAndArgCount & 0xFF];
+		if (0 == receiver) {
```

Comment:
Please use NULL rather than 0 for j9object_t.

Reply:
fixed